srcpkgs/volta/template

```diff
@@ -0,0 +1,77 @@
+# Template file for 'volta'
+pkgname=volta
+version=0.7.1
+revision=2
```

Comment:
New packages should start at revision 1.

Reply:
fixed.